sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobClient.java

```diff
@@ -392,28 +399,28 @@ public ServiceCall downloadWithHeaders(String containerName,
             callback);
     }
 
-    public void stageBlock(String containerName,
+    public BlockBlobsStageBlockResponse stageBlock(String containerName,
                            String blobName,
                            String base64BlockId,
-                           byte[] content,
+                           byte[] blockContent,
                            byte[] contentMd5) {
-        this.storageBlobServiceClient.stageBlock(containerName,
+        return this.storageBlobServiceClient.stageBlock(containerName,
             blobName,
             base64BlockId,
-            content,
+            blockContent,
             contentMd5);
     }
 
     public ServiceCall stageBlock(String containerName,
                                   String blobName,
                                   String base64BlockId,
-                                  byte[] body,
+                                  byte[] blockContent,
                                   byte[] contentMd5,
-                                  Callback<Void> callback) {
+                                  Callback<BlockBlobsStageBlockResponse> callback) {
```

Comment:
ref the previous comment, this is the async variant with "simplest" signature, and should be producing value in the Response (i.e. void) not the Response.

Reply:
So we would make this accept a `Callback<Void>` instead, right?

Reply:
right, I see you already made this change, sorry for delay.